index.html

```diff
@@ -41,7 +41,7 @@ <h1 class="logo">
             </nav>
             <div class="tools">
                 <form class="search">
-                    <button type="submit" class="search__btn">
+                    <button type="submit" method="post" class="search__btn">
```

Comment:
Suggestion
I think for search we can use method `get`

Reply:
Done